site/docs/recipes-api.md

```diff
@@ -82,27 +82,27 @@ document.write(`
 
 Your recipe configuration can also make use of existing variables, classes and styles.
 
-For example, you can use your `atoms` function from [Sprinkles.](https://github.com/seek-oss/vanilla-extract/tree/master/packages/sprinkles)
+For example, you can pass in the result of your [sprinkles](/documentation/sprinkles-api) function directly.
```

Comment:
```suggestion
For example, you can pass in the result of your [`sprinkles`](/documentation/sprinkles-api) function directly.
```

Reply:
I'll push this separately, the site doesn't support inline code as links very well.